specifyweb/frontend/js_src/lib/localization/common.ts

```diff
@@ -119,10 +119,9 @@ export const commonText = createDictionary({
   },
   delete: {
     'en-us': 'Delete',
-    'ru-ru': 'Конец',
     'es-es': 'Borrar',
-    'fr-fr': 'Fin',
-    'uk-ua': 'Кінець',
+    'fr-fr': 'Supprimer',
+    'uk-ua': 'видалити',
```

Comment:
(captialize)

```suggestion
    'uk-ua': 'Видалити',
```

I have no idea why Google Translate for UK and RU, when given a capitlized word, returns a non-capitalized - doesn't do it for other lanugages